packages/components/addon/components/hds/stepper/indicator/task.hbs

```diff
@@ -0,0 +1,11 @@
+<div class={{this.classNames}} ...attributes role="listitem" aria-label={{@status}}>
+  {{#if (eq @status "incomplete")}}
+    <FlightIcon class="hds-stepper-indicator-task__icon" @name="circle" @size="16" />
+  {{else if (eq @status "progress")}}
+    <FlightIcon class="hds-stepper-indicator-task__icon" @name="circle-half" @size="16" />
+  {{else if (eq @status "processing")}}
+    <FlightIcon class="hds-stepper-indicator-task__icon" @name="loading" @size="16" />
+  {{else if (eq @status "complete")}}
+    <FlightIcon class="hds-stepper-indicator-task__icon" @name="check-circle" @size="16" />
+  {{/if}}
```

Comment:
[non-blocker/suggestion] since the only difference between all these conditions is the icon name, what if we compute its value in the backing JavaScript class, similar to what is done with the `className`?

This would lead to this code:
```
<div class={{this.classNames}} ...attributes role="listitem" aria-label={{@status}}>
  <FlightIcon class="hds-stepper-indicator-task__icon" @name=={{this.iconName}} @size="16" />
</div>
```
which is much easier to read, and reduce the risk of bugs.